modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/util/ConfigurationModelToDDMFormValuesConverter.java

```diff
@@ -201,6 +203,31 @@ else if (type.equals(DDMFormFieldType.SELECT)) {
 		ddmFormFieldValue.setValue(localizedValue);
 	}
 
+	private void _validateDDMFormValuesWithOverrideConfiguration(
```

Comment:
I would rename to:

1.  _validateDDMFormValuesWithConfigurationOverrideProperties()
2. _validateDDMFormValuesWithOverrideProperties()